synapse/config/tls.py

```diff
@@ -143,31 +142,20 @@ def read_certificate_from_disk(self) -> None:
 
     def generate_config_section(
         self,
-        config_dir_path: str,
-        data_dir_path: str,
-        server_name: str,
         tls_certificate_path: Optional[str],
         tls_private_key_path: Optional[str],
         **kwargs: Any,
     ) -> str:
         """If the TLS paths are not specified the default will be certs in the
         config directory"""
 
-        base_key_name = os.path.join(config_dir_path, server_name)
-
         if bool(tls_certificate_path) != bool(tls_private_key_path):
             raise ConfigError(
                 "Please specify both a cert path and a key path or neither."
             )
 
         if tls_certificate_path and tls_private_key_path:
-            return (
-                """\
-            tls_certificate_path: "%(tls_certificate_path)s"
-            tls_private_key_path: "%(tls_private_key_path)s"
-            """
-                % locals()
-            )
+            return f"tls_certificate_path: {tls_certificate_path}\ntls_private_key_path: {tls_private_key_path}"
```

Comment:
Does this work? If so, I think it might be neater

```suggestion
            return f"""\
                tls_certificate_path: {tls_certificate_path}
                tls_private_key_path: {tls_private_key_path}
            """

Reply:
It does work, I have changed it.